musicFunction.js

```diff
@@ -0,0 +1,52 @@
+var original;
+var i = 0;
```

Comment:
Try to use const/let instead of var. Js convention is phasing out var. Variables declared with var are not block scoped (although they are function scoped), while with let and const they are.

Reply:
Thats really good to know thank you 👍